docs/airgap.md

```diff
@@ -0,0 +1,68 @@
+# Air gap scenario
+
+In some situations you may want to sync two Helm chart repositories without direct connectivity between them. 
+charts-syncer support this scenario via intermediate chart bundles.
+
+An intermediate chart bundle is just a tarball containing the original chart code plus the container images needed to deploy the chart.
```

Comment:
I'd add an additional paragraph here, something like.

`This intermediate Chart Bundle enables a two-step process where its content will be used to relocate the container images and the rewritten Helm Chart without contacting the source Container images registry or Helm repository. Making the relocation process suitable for disconnected, air-gap target environments`